src/nvim/decoration_defs.h

```diff
@@ -59,11 +59,7 @@ typedef struct {
   uint16_t flags;
   DecorPriority priority;
   int hl_id;  // if sign: highlight of sign text
-  // TODO(bfredl): Later signs should use sc[2] as well.
-  union {
-    char *ptr;  // sign
-    schar_T sc[2];  // conceal text (only sc[0] used)
-  } text;
+  schar_T text[2];  // conceal text only uses sc[0] used
```

Comment:
```suggestion
  schar_T text[SIGN_WIDTH];  // conceal text only uses sc[0
```

Reply:
Missed this one, perhaps intentionally, but the `used` typo is still there :)
There are still a few other usages of `2` which may be replaced by `SIGN_WIDTH`, but I'll leave that up to you.